Demo/Application/Features/PaymentButtonViewController.swift

```diff
@@ -2,52 +2,105 @@ import UIKit
 import SwiftUI
 import BraintreeUIComponents
 import BraintreeVenmo
+import BraintreePayPal
 
 class PaymentButtonViewController: PaymentButtonBaseViewController {
 
-    private var hostingController: UIHostingController<VenmoButton>?
-    private var selectedColor: VenmoButtonColor = .blue
+    private var hostingVenmoController: UIHostingController<VenmoButton>?
+    private var hostingPayPalController: UIHostingController<PayPalButton>?
+    private var selectedVenmoColor: VenmoButtonColor = .blue
+    private var selectedPayPalColor: PayPalButtonColor = .blue
 
     override func viewDidLoad() {
         super.viewDidLoad()
-        
+
         title = "Payment Buttons"
         view.backgroundColor = .systemBackground
 
-        setupColorSelector()
         setupVenmoButton()
+        setupPayPalButton()
     }
-    
-    private func setupColorSelector() {
-        let colorSegmentedControl = UISegmentedControl(items: ["Blue", "Black", "White"])
-        colorSegmentedControl.selectedSegmentIndex = 0
-        colorSegmentedControl.addTarget(self, action: #selector(colorChanged(_:)), for: .valueChanged)
-        colorSegmentedControl.translatesAutoresizingMaskIntoConstraints = false
-        
-        view.addSubview(colorSegmentedControl)
-        
-        NSLayoutConstraint.activate([
-            colorSegmentedControl.centerXAnchor.constraint(equalTo: view.centerXAnchor),
-            colorSegmentedControl.centerYAnchor.constraint(equalTo: view.centerYAnchor, constant: -80),
-            colorSegmentedControl.widthAnchor.constraint(equalToConstant: 250)
-        ])
+
+    private func colorChange(for buttonType: PaymentButtonType, selectedIndex: Int) {
+        switch buttonType {
+        case .venmo:
+            switch selectedIndex {
+            case 0: selectedVenmoColor = .blue
+            case 1: selectedVenmoColor = .black
+            case 2: selectedVenmoColor = .white
+            default: selectedVenmoColor = .blue
+            }
+            setupVenmoButton()
+        case .paypal:
+            switch selectedIndex {
+            case 0: selectedPayPalColor = .blue
+            case 1: selectedPayPalColor = .black
+            case 2: selectedPayPalColor = .white
+            default: selectedPayPalColor = .blue
+            }
```

Comment:
small cleanup suggestion:

```suggestion
            case .venmo:
            selectedVenmoColor = switch selectedIndex {
            case 0: .blue
            case 1: .black
            case 2: .white
            default: .blue
            }
            
            setupVenmoButton()
        case .paypal:
            selectedPayPalColor = switch selectedIndex {
            case 0: .blue
            case 1: .black
            case 2: .white
            default: .blue
            }
            
```